### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by twisterpilot**
- [DEMO LINK](https://twisterpilot.github.io/layout_calendar/)
- [TEST REPORT LINK] no test

- [x] Changing 'month-lengh' and 'start-day' modifier in the code element reflects in changing calendar layout
- [x] Each day has no modifiers, only class (eg. calendar__day)
- [x] All Typical Mistakes from BEM lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/var.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+$gap-size: 1px;
+$border-size: 1px;
+$week-days-position: (
+  "mon": 0,
+  "tue": ($cell-size + $gap-size),
+  "wed": 2 * ($cell-size + $gap-size),
+  "thu": 3 * ($cell-size + $gap-size),
+  "fri": 4 * ($cell-size + $gap-size),
+  "sat": 5 * ($cell-size + $gap-size),
+  "sun": 6 * ($cell-size + $gap-size)
```
Comment (maxim2310) [inline src/styles/var.scss]: would be better to move  ($cell-size + $gap-size) this part to loop directly for avoid repeating 
```suggestion
  "mon": 0,
  "tue": 1,
  "wed": 2 ,
  "thu": 3 ,
  "fri": 4 ,
  "sat": 5 ,
  "sun": 6 ,
```

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good work, but it will looks better if you center a calendar and fix one small thing in code

**Comment by twisterpilot** [comment]
Done, thanks!

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Please check demo link or try to deploy again.
Screenshot from 2023-12-01 14-05-23

**Pull request closed without merging**